backend/prisma/schema.prisma

```diff
@@ -28,8 +28,8 @@ model CodeGroup {
   id                     Int           @id @default(autoincrement())
   groupName              String
   imgUrl                 String?
-  preferred_category     String?
   preferred_availability String?
+  category               Interest[]
```

Comment:
For simplicity you could make so that a group only has one Interest, not a list of Interests. Up to you